### PR TITLE
entries: add new SppeedP and Hover Defender homebrew

### PR DESCRIPTION
**Pull request by kldobrev**
Added new entries

### CODE REVIEW

**Comment by avivace** [comment]
Can you correct Sport -> Sports ?

File: entries/speedp/game.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "platform": "GB",
+    "repository": "https://github.com/kldobrev/SpeedP",
+    "screenshots": [
+        "SpeedPScrn1.png",
+        "SpeedPScrn2.png"
+    ],
+    "slug": "speedp",
+    "tags": [
+        "Sport"
```
Comment (avivace) [inline entries/speedp/game.json:19]:
```suggestion
        "Sports"
```

**Review by avivace — CHANGES_REQUESTED** [review]

**Comment by kldobrev** [comment]
The tag is updated.

**Review by avivace — APPROVED** [review]

**Comment by kldobrev** [comment]
Hello and thank you for merging my commit. 
After adding the last 2 entries to the database, I tried searching for them in the Homebrew Hub, but I did not get any results. 
I read in the description that the changes should propagate automatically to it. 
Is there something else I need to do to make these entries visible?
Please advise.

**Comment by avivace** [comment]
> Hello and thank you for merging my commit. After adding the last 2 entries to the database, I tried searching for them in the Homebrew Hub, but I did not get any results. I read in the description that the changes should propagate automatically to it. Is there something else I need to do to make these entries visible? Please advise.

Hey @kldobrev ! Nothing needed from you're side, I'll trigger a re-deployment in the next few days ;)

**Pull request merged**